search/indexing_api.py

```diff
@@ -263,8 +292,22 @@ def index_post_with_comments(post_id):
                 errors=errors
             ))
 
+    for alias in get_active_aliases([COMMENT_TYPE]):
+        _, errors = bulk(
+            conn,
+            sync_comments(serialize_bulk_comments(post)),
+            index=alias,
+            doc_type=GLOBAL_DOC_TYPE,
+            # Adjust chunk size from 500 depending on environment variable
+            chunk_size=settings.ELASTICSEARCH_INDEXING_CHUNK_SIZE,
+        )
+        if len(errors) > 0:
+            raise ReindexException("Error during bulk insert: {errors}".format(
```

Comment:
Can you change the message here and above to say whether it was comment or post we're indexing?

Reply:
✅